task/orka-full/0.1/README.md

```diff
@@ -28,33 +26,31 @@ See also: [GCP-MacStadium Site-to-Site VPN](https://docs.macstadium.com/docs/goo
 
 Before you can use this `Task` in Tekton pipelines, you need to install it and the Orka configuration in your Kubernetes cluster.
 
-**Default namespace installation**
-
-To install in your Kubernetes cluster's `default` namespace, run the following command against your actual Orka API endpoint.
-
-```sh
-ORKA_API=http://10.221.188.100 ./install.sh --apply
-```
-
-To uninstall from the `default` namespace, run the script with the `-d` or `--delete` flag:
-
 ```sh
-./install.sh --delete
+kubectl apply --namespace=<namespace> -f orka-configuration.yaml
+kubectl apply --namespace=<namespace> -f https://raw.githubusercontent.com/tektoncd/catalog/orka-0.1/task/orka-full/0.1/orka-full.yaml
 ```
 
-**Custom namespace installation**
+Omit `--namespace` if installing in the `default` namespace.
 
-To install in a custom namespace, run the following command against your preferred namespace and your actual Orka API endpoint:
+You can use the following sample `orka-configuration.yaml`. Make sure to provide the correct Orka API endpoint for your Orka environment.
 
-```sh
-NAMESPACE=tekton-orka ORKA_API=http://10.221.188.100 ./install.sh --apply
+```yaml
```

Comment:
Thought - we might want to put the YAML example before the `kubectl apply` command - at first glance I was a bit confused as to where/what `orka-configuration.yaml` is, and other folks might be confused at first as well. Same below with the credentials.